packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -33,6 +33,7 @@ const production = 'production'
 const development = 'development'
 const analyzeBundle = process.env.MOBIFY_ANALYZE === 'true'
 const mode = process.env.NODE_ENV === production ? production : development
+const INSPECT = process.env.WEBPACK_INSPECT
```

Comment:
```suggestion
const INSPECT = process.execArgv.some(arg => /^--inspect(?:-brk)?(?:$|=)/.test(arg))
```
This will detect `--inspect`, `--inspect=1234`, `--inspect-brk`, and `--inspect-brk=1234`, even if the server is started using something other than `pwa-kit-dev`.

ref: [debugging](https://nodejs.org/en/docs/guides/debugging-getting-started)
ref: [`execArgv`](https://nodejs.org/docs/latest-v18.x/api/process.html#processexecargv)